### ISSUE TITLE
Python 3 dependency?

### ISSUE THREAD

**Issue by VMatrix1900** [prabirshrestha/asyncomplete-ultisnips.vim#1]
From [Installing](https://github.com/prabirshrestha/asyncomplete-ultisnips.vim#installing), it looks like this plugin requires vim has `+python3` feature. Is that true?

**Comment by prabirshrestha** [comment]
Yes. asyncomplete-utlisnips uses vim script api from ultisnip which internally calls python. So you need to have that feature.

If you want in pure vimscript you can use https://github.com/prabirshrestha/asyncomplete-neosnippet.vim

**Comment by VMatrix1900** [comment]
Oh, I see. Ultisnip requires python/python3 feature. Actually the python dependency comes from Ultisnip. And Ultisnips will check if Python/Python3 is available. See https://github.com/SirVer/ultisnips/blob/423f264e753cec260b4f14455126e6db7ba429af/doc/UltiSnips.txt#L547
So we can safely remove the Python3 check and let the Ultisnips handle its own dependency.
